### ISSUE TITLE
train using keras/tensorflow

### ISSUE THREAD

**Issue by JohanSchott** [JohanSchott/apartment_prices#5]
Instead of using scipy's minimize function, and a python implementation of the cost function and its gradient, train using keras/tensorflow. This should give a speed-up of the training.
